bot/exts/discord/discord.py

```diff
@@ -36,6 +43,49 @@ async def avatar(self, inter: ApplicationCommandInteraction, user: User) -> None
 
         await inter.response.send_message(embed=embed)
 
+    @discord.sub_command()
+    async def server(
+        self,
+        inter: ApplicationCommandInteraction,
+        guild_id: Optional[Guild] = None,
+        invite: Optional[Invite] = None,
+    ) -> None:
+        """
+        Get the icon of a Discord server.
+
+        Parameters
+        ----------
+        guild_id: The ID of the Discord server
+        invite: The invite to the Discord server
+        """
+        match (guild_id, invite):
+            case (None, None):
+                guild = inter.guild
+                icon = guild.icon
```

Comment:
```suggestion
                icon = inter.guild.icon
```
I think this is explicit enough in this case. If you think so too.

Reply:
I did this intentionally because I refer to the guild later on ([line 82](https://github.com/gustavwilliam/branding-bot/pull/8/files#diff-eca361431c1efc923d23a0f9e25db8316f086f53f59f7b039d20b1a9ef386485R82)).

Reply:
Oh, sneaky